### DISCUSSION TITLE
create a fork from litegraph.js then use it in the project so it is under your control not 3rd party.

### DISCUSSION THREAD

**Discussion by omar92** [comfyanonymous/ComfyUI#405] [Ideas]
There are some bugs in litegraph.js also some features needs that litegraph.js get organized better 

so having a fork from it should make it much easier for the community to contribute in fixing it , and will be faster than having to wait for the original one to accept the changes

**Comment by WASasquatch** [comment]
Might be a good idea. LiteGraph looks largely abandoned and crucial features left out, and then naturally missing from ComfyUI

**Comment by comfyanonymous** [comment]
I went and pushed the comfyui specific litegraph changes to this branch: https://github.com/comfyanonymous/litegraph.js/tree/comfyui

I would still prefer it if changes were submitted upstream first though.
